### PR TITLE
sync-to-date docs

### PR DESCRIPTION
**Pull request by T-256**
Removed resolved and outdated docs

### CODE REVIEW

File: docs/vs-requests.md [hunk 1]
```diff
@@ -3,8 +3,6 @@
 Although we try our best to mimic the requests API, some functionality is not easy to implement and left out.
 Here are a list of known incompatibilities:
 
-- retries are not supported yet, tracked in [#24](https://github.com/yifeikong/curl_cffi/issues/24)
-- stream/iterate are not supported. tracked in [#26](https://github.com/yifeikong/curl_cffi/issues/26)
 - files are not supported.
-- session cookies may lost during redirects, tracked in [#55](https://github.com/yifeikong/curl_cffi/issues/55)
```
Comment (yifeikong) [inline docs/vs-requests.md:9]: About cookies, under some discouraged condition, it's still lost. Could you please add some information about [this](https://github.com/yifeikong/curl_cffi/issues/119#issuecomment-1737724223)?

Comment (T-256) [reply docs/vs-requests.md:9]: Done, I'd recommend reopen #55 to keep tracking of that issue.

File: docs/vs-requests.md [hunk 2]
```diff
@@ -3,8 +3,7 @@
 Although we try our best to mimic the requests API, some functionality is not easy to implement and left out.
 Here are a list of known incompatibilities:
 
-- retries are not supported yet, tracked in [#24](https://github.com/yifeikong/curl_cffi/issues/24)
-- stream/iterate are not supported. tracked in [#26](https://github.com/yifeikong/curl_cffi/issues/26)
 - files are not supported.
```
Comment (yifeikong) [inline docs/vs-requests.md:6]: Can you add tracking issues for this, too?

Comment (T-256) [inline docs/vs-requests.md:6]:
```suggestion
- files are not supported, tracked in [#105](https://github.com/yifeikong/curl_cffi/issues/105)
```

**Pull request merged**